yarn.lock

```diff
@@ -2377,6 +2377,11 @@ anymatch@^2.0.0:
     micromatch "^3.1.4"
     normalize-path "^2.1.1"
 
+app-root-path@^2.2.1:
```

Comment:
Usually, this is modified if a new version of a dependency or a new dependency is needed to implement your feature.

I don't think this is your case. If so, revert these changes.